src/TTextEdit.cpp

```diff
@@ -862,6 +863,29 @@ void TTextEdit::normaliseSelection()
     }
 }
 
+void TTextEdit::expandSelectionToWords() {
```

Comment:
We put the opening brace on the next line.

BTW if you have `clang-format` installed you will find that Qt Creator can be told to apply the style settings that we have defined (in the `./src/.clang-format` file in the source code) to the stuff you have edited and then get it largely to match up with which we *intend*(ed! 😉) to be used throughout our code...